.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release SPRAS on new tags
+# This action runs whenever a new tag matching vX.Y.Z is pushed to the repo.
+# Its job is to build the `spras` container and push it to DockerHub with the
+# 'latest' tag, and a tag matching the one supplied to GitHub.
+
+on:
+  push:
+    tags:
+      # run release on any version tag vX.Y.Z, e.g. v0.2.1
+      - v[0-9]+\.[0-9]+\.[0-9]+
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ vars.DOCKER_USERNAME }}
```

Comment:
It turns out I added a DockerHub personal access token to this repository in 2021 and forgot about it
![image](https://github.com/user-attachments/assets/64601979-a52f-4292-8da2-4610faee5007)

Reply:
Nice!